cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers.go

```diff
@@ -123,23 +180,22 @@ func TryLoadCertAndKeyFromDisk(pkiPath, name string) (*x509.Certificate, *rsa.Pr
 	case *ecdsa.PrivateKey:
```

Comment:
Consuming ecdsa keys is already supported throughout, afaik. Generation is fixed to RSA currently.